packages/generators/generators/lib/plops/api.js

```diff
@@ -3,6 +3,10 @@
 const { join } = require('path');
 const fs = require('fs-extra');
 const validateInput = require('./utils/validate-input');
+const getCtNamesPrompts = require('./utils/get-ct-names-prompts');
```

Comment:
can we move these to a prompts folder instead of utils ? 
Most of them just export prompts maybe we shouldn't call them getX when they aren't a function